Team9ProjectTests/src/com/indragie/comput301as1/test/ReceiptTests.java

```diff
@@ -0,0 +1,17 @@
+package com.indragie.comput301as1.test;
+
+import com.indragie.cmput301as1.ExpenseItemAddActivity;
+
+import android.test.ActivityInstrumentationTestCase2;
+
+public class ReceiptTests extends ActivityInstrumentationTestCase2<ExpenseItemAddActivity> {
+
+	public ReceiptTests(String name) {
```

Comment:
@mrthegreensky Good call, after moving the bitmap scaling code to a separate method (mentioned above) there should be a test that takes an image that is larger than the limit, scales it, then verifies to check that it is within limits. This will test the functionality of the scaling code at the same time.